arbeitszeit_flask/templates/company/remove_worker_from_company.html

```diff
@@ -0,0 +1,42 @@
+{% extends "base.html" %}
+
+{% block navbar_start %}
+<div class="navbar-item">{{ gettext("Remove worker") }}</div>
+{% endblock %}
+
+{% block content %}
+
+<div class="section is-medium has-text-centered">
+  <div class="columns">
+    <div class="column"></div>
+    <div class="column is-5">
+      <div class="content">
+        <h1 class="title">{{ gettext("Remove worker") }}</h1>
+      </div>
+      <div>
+        <form method="post">
+          <input type="hidden" name="csrf_token" value="{{ csrf_token() }}">
+          <div class="field">
+            <label class="label">{{ gettext("Worker") }}</label>
+            <div class="control">
+              <div class="select">
+                <select name="worker">
+                  {% for worker in workers %}
+                  <option value="{{ worker.id }}">{{ worker.name }}</option>
```

Comment:
Should the worker ID perhaps be displayed along with the worker's name?  What if the company has two people with the same name (Max Mustermann)?